monai/bundle/scripts.py

```diff
@@ -168,17 +168,28 @@ def _get_git_release_url(repo_owner: str, repo_name: str, tag_name: str, filenam
     return f"https://github.com/{repo_owner}/{repo_name}/releases/download/{tag_name}/{filename}"
 
 
+def _get_ngc_base_url() -> str:
+    return "https://api.ngc.nvidia.com/v2/models/nvidia/monaitoolkit"
```

Comment:
I'd recommend string constants like this be represented as constant variables at the top of the file, rather than functions. It's clearer what these are this way and allows us to change the values temporarily for testing if ever we need to.

Reply:
Thanks @ericspod for the suggestion. I will change them as recommended.

Reply:
Fixed in [20c8424](https://github.com/Project-MONAI/MONAI/pull/7938/commits/20c84244f806df3f9e9b6d9aaf68e0e9d085489f)